bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/handler/EnOceanBaseSensorHandler.java

```diff
@@ -82,15 +86,20 @@ boolean validateConfig() {
         try {
             config.receivingEEPId.forEach(receivingEEP -> {
                 EEPType receivingEEPType = EEPType.getType(receivingEEP);
-                if (receivingEEPTypes.putIfAbsent(receivingEEPType.getRORG(), receivingEEPType) != null) {
+                @Nullable
```

Comment:
Are you sure? putIfAbsent can return null. If i remove the `@Nullable` the compiler outputs:
`Redundant null check: The variable existingKey cannot be null at this location`
But that is not true. My knowledge to these warnings and the compiler actually making code path's is limited. Why i don’t want is an NPE at runtime because the compiler left this code path out of the equation.